cmd/aws.go

```diff
@@ -55,13 +61,19 @@ func runAws(cmd *cobra.Command, args []string) error {
 
 	a, err := aws.New(c)
 	if err != nil {
-		return microerror.Mask(err)
+		fmt.Printf("Problem creating the AWS cleaner: %#v\n", err)
+		os.Exit(1)
 	}
 
 	err = a.Clean()
 	if err != nil {
-		return microerror.Mask(err)
+		// Print our collected errors
+		if errors, ok := err.(*errorcollection.ErrorCollection); ok {
+			fmt.Println("\nErrors:")
+			fmt.Println(errors.Dump())
```

Comment:
I would rather have `Dump` method doing the job of `Error`, so you would not need casting here.

Reply:
Do you mean I should let `ErrorCollection.Error()` do what `ErrorCollection.Dump()` is doing now? I thought about it, but Dump can create a pretty huge output. This way, `Error()` gives a very compact output that also works in a single line log entry.

Reply:
Yes that's what I meant.

The error collection is returned up the stack and only printed here, so this huge output would only be printed once here.
It would just be more practical to follow the `error` interface here, so printing the error could be achieved as usual without a need to cast it beforehand.